lit_gpt/prompts.py

```diff
@@ -19,6 +23,13 @@ def stop_tokens(self, tokenizer: "Tokenizer") -> Tuple[List[int], ...]:
     def from_name(cls, name: str) -> "PromptStyle":
         return prompt_styles[name]()
 
+    @classmethod
+    def from_config(cls, config: Config) -> "PromptStyle":
+        if config.name not in model_name_to_prompt_style:
+            # TODO: Warning here? Error?
+            return Default()
+        return prompt_styles[model_name_to_prompt_style[config.name]]()
```

Comment:
nit: I would split this into multiple lines to know which access caused a KeyError